<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>780</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(16,16,16,255);
border-radius:20px;</string>
   </property>
   <widget class="QLabel" name="LockImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(0,255,0,255);
align:center</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="AdminEditSup">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#AdminEditSup{
background-color:rgba(0,240,100,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#AdminEditSup:pressed{
background-color:rgba(0,255,150,255);
}
</string>
    </property>
    <property name="text">
     <string>Edit Suppliers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AdminEditPro">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#AdminEditPro{
background-color:rgba(0,240,100,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#AdminEditPro:pressed{
background-color:rgba(0,255,150,255);
}
</string>
    </property>
    <property name="text">
     <string>Edit Products</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AdminEditCus">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#AdminEditCus{
background-color:rgba(0,240,100,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#AdminEditCus:pressed{
background-color:rgba(0,255,150,255);
}
</string>
    </property>
    <property name="text">
     <string>Edit Customers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AdminPanLogOff">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#AdminPanLogOff{
background-color:rgba(255,125,0,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#AdminPanLogOff:pressed{
background-color:rgba(255,150,0,255);
}
</string>
    </property>
    <property name="text">
     <string> Log OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Close">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Close{
background-color:rgba(240,0,0,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#Close:pressed{
background-color:rgba(255,0,0,255);
}
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
